inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_kernel_b_fs_zyx_fsv16_imad.cpp

```diff
@@ -422,11 +425,25 @@ bool Convolution_kernel_b_fs_zyx_fsv16_imad::Validate(const Params& params, cons
     }
 
     KernelData kd = KernelData::Default<convolution_params>(params);
-    convolution_params& newParams = *static_cast<convolution_params*>(kd.params.get());
+    convolution_params& conv_params = *static_cast<convolution_params*>(kd.params.get());
 
-    if (newParams.split != 1)
+    if (conv_params.split != 1)
         return false;
 
+    if (conv_params.quantization == QuantizationType::ASYMMETRIC_DATA_AND_WEIGHTS) {
+        if (conv_params.activations_zero_points.empty() || conv_params.weights_zero_points.empty())
+            return false;
+    } else if (conv_params.quantization == QuantizationType::ASYMMETRIC_DATA) {
```

Comment:
Here you should also check that `!conv_params.compensation.empty()` for both ASYMMETRIC_DATA and ASYMMETRIC_DATA_AND_WEIGHTS quantization types

Reply:
Fixed